modules/internal/ChapelArray.chpl

```diff
@@ -3263,7 +3263,9 @@ module ChapelArray {
       if !a.stridable && b.stridable then
         compilerError("cannot assign from a stridable domain to an unstridable domain without an explicit cast");
 
-    a._instance.dsiAssignDomain(b, lhsPrivate=false);
+    // distributions/robust/arithmetic/resizing/resizeFromOtherLocale.chpl
+    // fails if this uses b._instance instead of b._value
+    a._value.dsiAssignDomain(b, lhsPrivate=false);
```

Comment:
No, it's not always empty.  I added the following to chpl_assignDomainWithGetSetIndices:

```
if _isPrivatized(lhs) {
  writeln(lhs._arrs.size);
}
```

In the resizeFromOtherLocale.chpl test mentioned in this comment, `chpl_assignDomainWithGetSetIndices` is called 3 times with a privatized lhs, and the size of `lhs._arrs` in those 3 calls are 0, 1, 0.